RecoLocalTracker/SiPixelRecHits/interface/PixelCPEFastParamsHost.h

```diff
@@ -0,0 +1,51 @@
+#ifndef DataFormats_PixelCPEFastParams_interface_PixelCPEFastParamsHost_h
+#define DataFormats_PixelCPEFastParams_interface_PixelCPEFastParamsHost_h
+
+#include <alpaka/alpaka.hpp>
+
+#include "CondFormats/SiPixelTransient/interface/SiPixelGenError.h"
+#include "DataFormats/GeometrySurface/interface/SOARotation.h"
+#include "DataFormats/SiPixelClusterSoA/interface/ClusteringConstants.h"
+#include "DataFormats/TrackingRecHitSoA/interface/SiPixelHitStatus.h"
+#include "Geometry/CommonTopologies/interface/SimplePixelTopology.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/CopyToDevice.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/memory.h"
+#include "RecoLocalTracker/SiPixelRecHits/interface/PixelCPEGenericBase.h"
+#include "RecoLocalTracker/SiPixelRecHits/interface/pixelCPEforDevice.h"
+
+template <typename TrackerTraits>
+class PixelCPEFastParamsHost : public PixelCPEGenericBase {
```

Comment:
I think the host class needs to be made non-copyable too.

Reply:
Done.